src/main/java/xyz/vegaone/easytracking/controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package xyz.vegaone.easytracking.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import xyz.vegaone.easytracking.dto.User;
+import xyz.vegaone.easytracking.service.UserService;
+
+@RestController
+@RequestMapping(value = "/api/user")
+@Slf4j
+public class UserController {
+
+    @Autowired
+    private UserService userService;
```

Comment:
we've switched to constructor based injection, could you please inject the bean through the constructor?